waik/PXELinux.erb

```diff
@@ -2,6 +2,9 @@
 kind: PXELinux
 name: WAIK default PXELinux
 %>
-default winPE
-label winPE
-kernel /Boot/startrom.0
+default wimboot
+label wimboot
+    menu default
+     menu label Wimboot via pxelinux
```

Comment:
I recall PXELinux had some issue with two menu items, but not sure of the details. Does this work?